pyqtgraph/graphicsItems/InfiniteLine.py

```diff
@@ -607,6 +609,7 @@ def hoverEvent(self, ev):
             ev.acceptDrags(QtCore.Qt.LeftButton)
 
     def viewTransformChanged(self):
+        GraphicsItem.viewTransformChanged(self)
```

Comment:
I did not look into the code (example), but normally the `TextItem.viewTransformChanged` should do the job. Did you check if the order is important?